.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj
+* @dav3r @felddy @jasonodoom @jsf9k @king-alexander @mcdonnnj
```

Comment:
Feel free to add yourself as a codeowner below as well!

Reply:
Alphabetization is next to godliness ([4d769b1](https://github.com/cisagov/ansible-role-admiral/pull/3/commits/4d769b187c462b69d2252658765fef446e59240b)).